feature2/anotherfile.txt

```diff
@@ -0,0 +1,2 @@
+Added one more file from feature2 branch locally
```

Comment:
It was indeed a good addition to this file

Reply:
Thank you Dinesh!